README.md

```diff
@@ -64,6 +64,7 @@ Core lib's Maven group ID is `io.apimatic`, and its artifact ID is `core`.
 | [`CoreJsonObject`](./src/main/java/io/apimatic/core/utilities/CoreJsonObject.java)                                               | Wrapper class for JSON object                                                                                                                                                                                                                         |
 | [`CoreJsonValue`](./src/main/java/io/apimatic/core/utilities/CoreJsonValue.java)                                                 | Wrapper class for JSON value                                                                                                                                                                                                                          |
 | [`TestHelper`](./src/main/java/io/apimatic/core/utilities/TestHelper.java)                                                       | Contains utility methods for comparing objects, arrays and files                                                                                                                                                                                      |
+| [`ConversionHelper`](./src/main/java/io/apimatic/core/utilities/ConversionHelper.java)                                           | A Helper class for the coversion of type (provided as function) for all structures (array, map, array of map, n-dimensional arrays etc) supported in the SDK.                                                                                         |
 
```

Comment:
Do we need to add AdditionalProperties in the README.md?

Reply:
Yes, the `ConversionHelper` class should be added to the README.md for consistency, as it handles type conversions for various structures and other helper classes are already documented.

Reply:
`AdditionalProperties`  has also been added to README.md